server/routes/juror-management/juror-record/confirm-identity/confirm-identity.controller.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable strict */
+
+const _ = require('lodash');
+const validate = require('validate.js');
+const idTypeValidator = require('../../../../config/validation/confirm-identity');
+const { systemCodesDAO, confirmIdentityDAO } = require('../../../../objects');
+const { Logger } = require('../../../../components/logger');
+
+module.exports.getConfirmIdentity = function(app) {
+  return async function(req, res) {
+    const { jurorNumber } = req.params;
+
+    let idCheckCodes = [{ value: '', text: 'Select ID type' }];
+
+    try {
+      (await systemCodesDAO.get(app, req, 'ID_CHECK')).reduce((acc, code) => {
```

Comment:
`array.reduce` returns it's value at the end - I think this is working because arrays are call-by-reference, but this could lead to bugs down the line. I'd rather we in-lined the starting condition into the reduce, and returned into `idCheckCodes`

Reply:
updated